plugins/ruby-foreman-tasks/debian/gem.list

```diff
@@ -1,5 +1,5 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman-tasks-0.17.0.gem"
+GEMS="https://rubygems.org/downloads/foreman-tasks-0.17.1.gem"
 GEMS="$GEMS https://rubygems.org/downloads/foreman-tasks-core-0.3.1.gem"
 GEMS="$GEMS https://rubygems.org/downloads/parse-cron-0.1.4.gem"
 GEMS="$GEMS https://rubygems.org/downloads/sinatra-2.0.5.gem"
```

Comment:
Could all these ones get updated to their latest versions, too?

Reply:
Done